package.json

```diff
@@ -62,6 +62,7 @@
     "babel-plugin-external-helpers": "^6.22.0",
     "babel-plugin-module-resolver": "^3.0.0",
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
+    "babel-polyfill": "^6.26.0",
```

Comment:
Is it possible to do this without `babel-polyfill`? Modern versions of Node support async / await out of the box.

Reply:
After running examples I got `regeneratorRuntime` error.
I tried to solve it by testing all options from [babel docs ](https://babeljs.io/docs/plugins/transform-regenerator/) and finally this one worked.

Reply:
babel-polyfill shouldn't be used in a library, so we should find another way to do this. Have you tried babel-preset-env?

Reply:
The other option would be to not use Babel on anything in src/server and write it without ES5 features. I can take a look at this later.